### ISSUE TITLE
Make it clear that BIDS datasets should already be scaled

### ISSUE THREAD

**Issue by tsalo** [PennLINC/aslprep#293]
## Summary

At minimum, our test datasets (mostly) do not contain scaled M0 scans, which means that the `--m0_scale` parameter is required to produce reasonable CBF estimates. However, BIDS is actually pretty clear that [the dataset MUST contain scaled data](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#scaling), which may not be apparent to dataset curators and ASLPrep users.

Basically, I want to document that the datasets should already be scaled, _and_ make it clear that `--m0_scale` is necessary when they aren't already.
